### DISCUSSION TITLE
Explore to Sunbird-RC page from Sunbird-RC pages

### DISCUSSION THREAD

**Discussion by jagadishbabu** [Sunbird-RC/community#16] [General]
Sunbird-RC team.

I am unable to explore to Sunbird-RC pages from Sunbird.org. Am I missing any links?  can the team ensure that Sunbird-RC pages/repo links are explorable from the main page?

-jagadish

**Comment by pramodkvarma** [comment]
We will be linking the https://www.sunbirdrc.dev documentation site from sunbird.org soon. Please stay tuned.

**Reply by parthlawate** [reply]
This is done @jagadishbabu @pramodkvarma https://sunbird.org/projects

**Reply by jagadishbabu** [reply]
@parthlawate thank you. we can close this thread
